udf/agent/server.go

```diff
@@ -74,7 +74,7 @@ func (s *Server) StopOnSignals(signals ...os.Signal) {
 	s.wg.Add(1)
 	s.mu.Unlock()
 
-	c := make(chan os.Signal)
+	c := make(chan os.Signal, 1)
```

Comment:
Why?

Reply:
It failed `go vet`, reasoning explained in the release notes [here](https://golang.org/doc/go1.17#vet).